lisa/util/process.py

```diff
@@ -2,12 +2,21 @@
 import os
 import shlex
 import subprocess
+import time
 from threading import Thread
-from typing import Dict, Optional, cast
+from timeit import default_timer as timer
+from types import TracebackType
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Type, cast
 
-from psutil import Process as psutilProcess
+import psutil
```

Comment:
Have you looked into [pexpect](https://github.com/pexpect/pexpect)? It's what `sosreport` uses to great satisfaction.

Reply:
It looks better, one concern is that doesn't support windows well. I put it in backlog, we can check it later.
https://pexpect.readthedocs.io/en/stable/overview.html#pexpect-on-windows